example/cdp.js

```diff
@@ -0,0 +1,68 @@
+const _ = require("lodash");
+
+const kavaUtils = require("../src/utils").utils;
+const KavaClient = require("../src/client").KavaClient;
+
+const KAVA_API_TESTNET_6000_INTERNAL = "http://54.196.2.124:1317";
+const kavaAddress = "kava1g0qywkx6mt5jmvefv6hs7c7h333qas5ks63a6t";
+const kavaMnemonic =
+  "lecture draw addict sea prefer erupt army someone album liquid sadness manual fence vintage obey shrimp figure retreat kick refuse verify alien east brand";
+
+const BNB_CONVERSION_FACTOR = 10 ** 8;
```

Comment:
let's create an issue to add a client method for `getConversionFactor`, I've noticed that I'm either hard coding it or reusing a lot of code to get it - would be nicer to have on the client.

Reply:
https://github.com/Kava-Labs/javascript-sdk/issues/21